pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -8,6 +8,7 @@ if [ $# -eq 0 ]; then
 fi
 
 pkgs=$1
+tmpfile=$(mktemp /tmp/nuget-to-nix.XXXXXX)
```

Comment:
```suggestion
tmpfile=$(mktemp /tmp/nuget-to-nix.XXXXXX)
trap 'rm -f "$tmpfile" EXIT
```
This is needed so the temporary file also gets deleted if the scripts exits early (due to `set -e`)